src/techjoomla/tjnotifications/tjnotifications.php

```diff
@@ -240,4 +243,52 @@ public static function getTags($data_template)
 		//  $matches[0]= stores tag like {doner.name} and $matches[1] stores doner.name. Explode it and make it doner->name
 			return $matches;
 	}
+
+	/**
+	 * Method to get Sample data for email template.
+	 *
+	 * @return  string  $body
+	 *
+	 * @since 1.0
+	 */
+	public static function getSampleBodyData()
+	{
+		$input = Factory::getApplication()->input;
+		$id = $input->get('id');
```

Comment:
@punambaravkar pass Id as a parameter to this function. Also Why this is in the library, move this to notification model